### ISSUE TITLE
Backup with different frequency

### ISSUE THREAD

**Issue by carlo-github** [Yenthe666/auto_backup#149]
I would like to create a backup every 4 hours and save it in 1 server via sftp and a backup 1 time per week to be store in another server via sftp. Can I do that?

**Comment by Yenthe666** [comment]
Hi @carlo-github,

Partially. You can create two backup records that point to two different FTP servers. The scheduler itself goes off for every single backup record though so if you'd like to do one backup every 4 hours to location A and one backup every week to location B you'll need to add extra logic to the filter/search of the scheduler. Another option is an extra field on the backup record which specifies the interval rate for example.

TLDR: Yes to backup to two locations, no to different time intervals without some custom dev.
